Sources/web3swift/Browser/Bridge.swift

```diff
@@ -6,6 +6,7 @@
 //  Copyright © 2017 Samaritan. All rights reserved.
 //
 
+#if !os(Linux)
 import WebKit
```

Comment:
Guess this file should be excluded from the sources on the Linux platform in a manner that it is now for macOS one. Means it should be ignored in `Package.swift`.